pkg/service/common.go

```diff
@@ -48,8 +48,8 @@ func SanitizeAnalyzerNode(analyzerNode *Analyzer) {
 	analyzerNode.NodeType = NodeTypeAnalyzerNode
 }
 
-func SanitizePackageNode(pkgNode *PackageNode, dbpkgNode *PackageNode) {
-	pkgNode.Uid = dbpkgNode.Uid
-	pkgNode.Session = dbpkgNode.Session
-	pkgNode.NodeType = NodeTypePackageNode
+func SanitizePackageNode(pkgNode *PackageNode) {
+	for _, info := range pkgNode.AdditionalInfo {
```

Comment:
This is not equivalent. Why did you deleted the checks for session, nodetype etc?

Reply:
Have a look now at the commit message.